packages/esm-patient-medications-app/src/add-drug-order/add-drug-order.workspace.tsx

```diff
@@ -54,9 +58,24 @@ export default function AddDrugOrderWorkspace({ order: initialOrder, closeWorksp
       closeWorkspace();
       launchPatientWorkspace('order-basket');
     },
-    [orders, setOrders, closeWorkspace, session.currentProvider.uuid],
+    [orders, setOrders, closeWorkspace, session?.currentProvider?.uuid],
   );
 
+  if (session.currentProvider == null) {
+    return (
+      <div className={styles.tileContainer}>
+        <Tile className={styles.tile}>
+          <div className={styles.tileContent}>
+            <p className={styles.content}>
+              {t('providerNotAllowed', 'User provider is not allowed to perform this action')}
```

Comment:
@ibacher how would you word this better?

Also, @gabriel090, the established convention here is to render one of the [notification types](https://zeroheight.com/23a080e38/p/683580-notifications/t/60ea14) described in the notifications guide.

Reply:
"The current user is not setup as a provider, and so cannot place drug orders. If you feel that this is in error, please contact your system administrator for assistance."

Reply:
Also, I would use something like the same text for lab orders. This _should_ be a toast notification.